<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.0.0-beta.1 for Hugo"><meta name=author content="Theo Sanderson"><meta name=description content="Code behind this analysis: https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data
The ONS infection survey has come out and there has been a lot of discussion on the apparent decrease in proportion of &ldquo;new variant compatible&rdquo; cases."><link rel=alternate hreflang=en-us href=/post/2021-01-22-ons-data/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#1565c0"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark media=print onload="this.media='all'" disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA==" crossorigin=anonymous media=print onload="this.media='all'"><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload="this.media='all'"><link rel=stylesheet href=/css/wowchemy.a2dda909fb8228e17412d06b4423e0a6.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-65298-19"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-65298-19',{});document.addEventListener('click',onClickCallback,false);</script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_192x192_fill_lanczos_center_2.png><link rel=canonical href=/post/2021-01-22-ons-data/><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@theosanderson"><meta property="twitter:creator" content="@theosanderson"><meta property="og:site_name" content="Theo Sanderson"><meta property="og:url" content="/post/2021-01-22-ons-data/"><meta property="og:title" content="New-variant compatibility in the ONS infection survey | Theo Sanderson"><meta property="og:description" content="Code behind this analysis: https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data
The ONS infection survey has come out and there has been a lot of discussion on the apparent decrease in proportion of &ldquo;new variant compatible&rdquo; cases."><meta property="og:image" content="/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_512x512_fill_lanczos_center_2.png"><meta property="twitter:image" content="/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2021-01-22T14:18:37+00:00"><meta property="article:modified_time" content="2021-01-22T14:58:37+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","mainEntityOfPage":{"@type":"WebPage","@id":"/post/2021-01-22-ons-data/"},"headline":"New-variant compatibility in the ONS infection survey","datePublished":"2021-01-22T14:18:37Z","dateModified":"2021-01-22T14:58:37Z","author":{"@type":"Person","name":"Theo Sanderson"},"publisher":{"@type":"Organization","name":"Theo Sanderson","logo":{"@type":"ImageObject","url":"/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_192x192_fill_lanczos_center_2.png"}},"description":"Code behind this analysis: https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data\nThe ONS infection survey has come out and there has been a lot of discussion on the apparent decrease in proportion of \u0026ldquo;new variant compatible\u0026rdquo; cases."}</script><title>New-variant compatibility in the ONS infection survey | Theo Sanderson</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper><script src=/js/wowchemy-init.min.7264cf0eba3b66951b36da7d2cecf9c5.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Theo Sanderson</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Theo Sanderson</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Posts</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><article class=article><div class="article-container pt-3"><h1>New-variant compatibility in the ONS infection survey</h1><div class=article-metadata><span class=article-date>Last updated on
Jan 22, 2021</span></div></div><link rel=stylesheet href=/theo.css><div class=article-container><div class=article-style><a class=viewsource href=https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data/index.md>View source</a><p><small>Code behind this analysis: <a href=https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data class=uri><a href=https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data>https://github.com/theosanderson/theo.io/tree/master/content/post/2021-01-22-ons-data</a></a></small></p><p>The ONS infection survey has come out and there has been a lot of discussion on the apparent decrease in proportion of &ldquo;new variant compatible&rdquo; cases.</p><p><img src=ons.png alt></p><p>To try to understand these patterns we need to go into a bit more detail. How are &ldquo;new variant compatible&rdquo; cases defined?</p><p>The TaqPath tests that produce this data amplify parts of three genes in the SARS-CoV2 genome:</p><ul><li>The N gene</li><li>ORF1ab</li><li>The S gene</li></ul><p>We know that B.1.1.7 often gives complete loss of the S-gene amplicon (data from Portugal suggest it doesn&rsquo;t <a href=https://virological.org/t/tracking-sars-cov-2-voc-202012-01-lineage-b-1-1-7-dissemination-in-portugal-insights-from-nationwide-rt-pcr-spike-gene-drop-out-data/600 target=_blank rel=noopener>always</a>).</p><p>When B.1.1.7 was emerging the ONS defined a fairly conservative definition of &ldquo;new variant compatible&rdquo; tests. They said that these must be positive for N, <em>and</em> ORF1AB, but not for S. That is what we would expect to see for B.1.1.7 when there a lot of virions in the sample. But if a sample has a low number of virions, one or other of these genes might randomly drop below the detection threshold. Fortunately the ONS also report the data split out by each of the possible amplicon combinations, so we can examine this.</p><p>First let&rsquo;s convince ourselves that, irrespective of B.1.1.7, dropouts of amplicons can occur for various reasons. Let&rsquo;s look at the distribution of amplicons period of time in September, before B.1.1.7 had really emerged.</p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=nv>subset</span> <span class=o>=</span> <span class=nv>data</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>week</span><span class=o>==</span><span class=s>"2020-09-21"</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf>mutate</span><span class=o>(</span>AmpliconType <span class=o>=</span> <span class=nf>case_when</span><span class=o>(</span>
 <span class=nv>Amplicons</span> <span class=o>%in%</span> <span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=s>"N only"</span>,<span class=s>"N+S"</span>,<span class=s>"OR only"</span>,<span class=s>"OR+S"</span>,<span class=s>"S only"</span><span class=o>)</span> <span class=o>~</span> <span class=s>"Random dropout"</span>,
  <span class=nv>Amplicons</span> <span class=o>==</span> <span class=s>"OR+N+S"</span> <span class=o>~</span> <span class=s>"Definite non-B.1.1.7"</span>,
  <span class=nv>Amplicons</span> <span class=o>==</span> <span class=s>"OR+N"</span> <span class=o>~</span> <span class=s>"Likely(?) B.1.1.7"</span>
<span class=o>)</span><span class=o>)</span>
                                                                                                
                                                                                                
<span class=nf>ggplot</span><span class=o>(</span><span class=nv>subset</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>AmpliconType</span>,y<span class=o>=</span><span class=nv>Count</span>,fill<span class=o>=</span><span class=nv>Amplicons</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_bar</span><span class=o>(</span>stat<span class=o>=</span><span class=s>"identity"</span><span class=o>)</span><span class=o>+</span><span class=nf>facet_wrap</span><span class=o>(</span><span class=o>~</span><span class=nv>Region</span><span class=o>)</span><span class=o>+</span><span class=nf>theme_bw</span><span class=o>(</span><span class=o>)</span><span class=o>+</span> <span class=nf>theme</span><span class=o>(</span>axis.text.x <span class=o>=</span> <span class=nf>element_text</span><span class=o>(</span>angle <span class=o>=</span> <span class=m>90</span>, vjust <span class=o>=</span> <span class=m>0.5</span>, hjust<span class=o>=</span><span class=m>1</span><span class=o>)</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-2-1.png width=700px style=display:block;margin:auto></p></div><p>I&rsquo;ve separated out the <code>OR+N</code> amplicon types, which the ONS considers new variant compatible. Let&rsquo;s ignore these for now. We can see that there is a substantial variation in the other amplicon groups. If there were no random dropout (and no other variants that compromise an assay), we would expect everything else to be <code>OR+N+S</code>. But in the South West these make up a minority of the total positives identified. Instead it seems that some of these amplicons are randomly dropping out there.</p><p>We can also see a lot of geographical heterogeneity in the level of this random dropout. What drives this? We also have the mean Ct values for each region so we can test the hypothesis that high Ct values (low number of virions), due to a higher proportion of infections being identified longer after the original infection, are responsible for the &ldquo;random dropout&rdquo;.</p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=c># Exclude suspected B.1.1.7 from the data</span>
<span class=nv>subset</span> <span class=o>=</span> <span class=nv>data</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>week</span><span class=o>==</span><span class=s>"2020-09-21"</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Amplicons</span><span class=o>!=</span><span class=s>"OR+S"</span><span class=o>)</span>

<span class=c>#Calculate the proportion with "no random dropout", then invert to get proportion of "random dropout"</span>
<span class=nv>subset</span> <span class=o>=</span><span class=nv>subset</span> <span class=o>%&gt;%</span> <span class=nf>group_by</span><span class=o>(</span><span class=nv>Region</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf>mutate</span><span class=o>(</span>proportion <span class=o>=</span> <span class=nv>Count</span><span class=o>/</span><span class=nf><a href=https://rdrr.io/r/base/sum.html>sum</a></span><span class=o>(</span><span class=nv>Count</span><span class=o>)</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Amplicons</span><span class=o>==</span><span class=s>"OR+N+S"</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf>mutate</span><span class=o>(</span>RandomDropout<span class=o>=</span><span class=m>1</span><span class=o>-</span><span class=nv>proportion</span><span class=o>)</span>

<span class=nv>subset_cts</span> <span class=o>=</span> <span class=nv>data_ct</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>week</span><span class=o>==</span><span class=s>"2020-09-21"</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span><span class=o>)</span>

<span class=nv>both</span> <span class=o>&lt;-</span> <span class=nf>inner_join</span><span class=o>(</span><span class=nv>subset</span>,<span class=nv>subset_cts</span><span class=o>)</span> 

<span class=c>#&gt; Joining, by = c("Region", "RegionType", "week")</span>


<span class=nf>ggplot</span><span class=o>(</span><span class=nv>both</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>Mean</span>,y<span class=o>=</span><span class=nv>RandomDropout</span>,label<span class=o>=</span><span class=nv>Region</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_point</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>labs</span><span class=o>(</span>x<span class=o>=</span><span class=s>"Mean Ct value"</span>,y<span class=o>=</span><span class=s>"Proportion of 'random dropout'"</span><span class=o>)</span><span class=o>+</span><span class=nf><a href=https://rdrr.io/pkg/ggrepel/man/geom_text_repel.html>geom_text_repel</a></span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>theme_bw</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span><span class=o>+</span><span class=nf>labs</span><span class=o>(</span><span class=s>"Relationship between Ct and levels of random dropout in September"</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-3-1.png width=700px style=display:block;margin:auto></p></div><p>Having convinced ourselves that random dropouts can occur at different levels due to differences in Ct values (and that Ct values <a href=https://www.medrxiv.org/content/10.1101/2020.10.08.20204222v1 target=_blank rel=noopener>can differ due to epidemiological dynamics</a>)), we can examine whether an increased rate of random dropouts in the N or ORF1ab genes (or both!) might cause an underestimation of B.1.1.7 prevalence when considering only <code>OR+N+S</code> as compatible with the new variant.</p><p>Now lets look at the trajectories of all amplicon groups over time in England.</p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=nv>data</span> <span class=o>=</span> <span class=nv>data</span> <span class=o>%&gt;%</span> <span class=nf>group_by</span><span class=o>(</span><span class=nv>Region</span>,<span class=nv>week</span><span class=o>)</span> <span class=o>%&gt;%</span> <span class=nf>mutate</span><span class=o>(</span>proportion <span class=o>=</span> <span class=nv>Count</span><span class=o>/</span><span class=nf><a href=https://rdrr.io/r/base/sum.html>sum</a></span><span class=o>(</span><span class=nv>Count</span><span class=o>)</span><span class=o>)</span> 

<span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Region</span><span class=o>==</span><span class=s>"England"</span><span class=o>)</span>, <span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>,y<span class=o>=</span><span class=nv>proportion</span>,color<span class=o>=</span><span class=nv>Amplicons</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_line</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>theme_bw</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-4-1.png width=700px style=display:block;margin:auto></p><pre class=chroma><code class=language-r data-lang=r>

<span class=nv>data</span><span class=o>$</span><span class=nv>Amplicons</span> <span class=o>=</span> <span class=nf><a href=https://rdrr.io/r/base/factor.html>factor</a></span><span class=o>(</span><span class=nf><a href=https://rdrr.io/r/base/character.html>as.character</a></span><span class=o>(</span><span class=nv>data</span><span class=o>$</span><span class=nv>Amplicons</span><span class=o>)</span>,levels <span class=o>=</span> <span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=s>"S only"</span>,<span class=s>"OR+S"</span>,<span class=s>"N+S"</span>,<span class=s>"OR+N+S"</span>,<span class=s>"N only"</span>,<span class=s>"OR only"</span>,<span class=s>"OR+N"</span><span class=o>)</span><span class=o>)</span>

<span class=nv>p</span><span class=o>&lt;-</span><span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Region</span><span class=o>==</span><span class=s>"England"</span><span class=o>)</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Amplicons</span><span class=o>!=</span><span class=s>"S only"</span><span class=o>)</span>, <span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>,y<span class=o>=</span><span class=nv>proportion</span>,fill<span class=o>=</span><span class=nv>Amplicons</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_area</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>theme_bw</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>expand<span class=o>=</span><span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=m>0</span>,<span class=m>0</span><span class=o>)</span>,labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span><span class=o>+</span><span class=nf>scale_fill_brewer</span><span class=o>(</span>type<span class=o>=</span><span class=s>"qual"</span>,palette<span class=o>=</span><span class=m>4</span><span class=o>)</span><span class=o>+</span><span class=nf>labs</span><span class=o>(</span>x<span class=o>=</span><span class=s>"Date"</span>,y<span class=o>=</span><span class=s>"Proportion"</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_x_date</span><span class=o>(</span>expand<span class=o>=</span><span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=m>0</span>,<span class=m>0</span><span class=o>)</span><span class=o>)</span>

<span class=nf>ggsave</span><span class=o>(</span><span class=s>"plot.pdf"</span>,width<span class=o>=</span><span class=m>5</span>,height<span class=o>=</span><span class=m>3</span><span class=o>)</span>
</code></pre></div><p>An annotated version of that plot: <img src=illustrated_plot-01.png alt></p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span><span class=o>)</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Amplicons</span><span class=o>!=</span><span class=s>"S only"</span><span class=o>)</span>, <span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>,y<span class=o>=</span><span class=nv>proportion</span>,fill<span class=o>=</span><span class=nv>Amplicons</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_area</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>theme_bw</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>expand<span class=o>=</span><span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=m>0</span>,<span class=m>0</span><span class=o>)</span>,labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span><span class=o>+</span><span class=nf>scale_fill_brewer</span><span class=o>(</span>type<span class=o>=</span><span class=s>"qual"</span>,palette<span class=o>=</span><span class=m>4</span><span class=o>)</span><span class=o>+</span><span class=nf>labs</span><span class=o>(</span>x<span class=o>=</span><span class=s>"Date"</span>,y<span class=o>=</span><span class=s>"Proportion"</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_x_date</span><span class=o>(</span>expand<span class=o>=</span><span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=m>0</span>,<span class=m>0</span><span class=o>)</span><span class=o>)</span> <span class=o>+</span><span class=nf>facet_wrap</span><span class=o>(</span><span class=o>~</span><span class=nv>Region</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-5-1.png width=700px style=display:block;margin:auto></p><pre class=chroma><code class=language-r data-lang=r>
<span class=nf>ggsave</span><span class=o>(</span><span class=s>"plotregion.png"</span>,width<span class=o>=</span><span class=m>7</span><span class=o>*</span><span class=m>1.2</span>,height<span class=o>=</span><span class=m>4</span><span class=o>*</span><span class=m>1.2</span><span class=o>)</span>
</code></pre></div><p>We can see a rise in <code>OR+N</code> over time. But this seems to tail off to at least horizontal at the end. Does this mean that B.1.1.7 is no longer increasing at the expense of other variants? Well I don&rsquo;t think we have evidence for this. <code>OR+N</code> may be horizontal, but <code>OR+N+S</code> seems likely to be decreasing faster. It is likely that this is due to a general change in the epidemic stage, as lockdown controls new infections and a higher proportion of old detections are detected.</p><p>Again if we want to look for an explanation we can investigate the Cts.</p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=nf>ggplot</span><span class=o>(</span><span class=nv>data_ct</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span><span class=o>)</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>, y<span class=o>=</span><span class=nv>Mean</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_line</span><span class=o>(</span><span class=o>)</span> <span class=o>+</span><span class=nf>labs</span><span class=o>(</span>title<span class=o>=</span><span class=s>"Regions"</span><span class=o>)</span><span class=o>+</span><span class=nf>facet_wrap</span><span class=o>(</span><span class=o>~</span><span class=nv>Region</span><span class=o>)</span><span class=o>+</span><span class=nf>theme_bw</span><span class=o>(</span><span class=o>)</span><span class=o>+</span><span class=nf>labs</span><span class=o>(</span>y<span class=o>=</span><span class=s>"Mean Ct value"</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-6-1.png width=700px style=display:block;margin:auto></p></div><p>In all regions we see an increase in mean Ct over January, which we&rsquo;d expect to cause more random dropouts, and reduce the number of both WT viruses that appear as <code>OR+N+S</code> and of B.1.1.7 viruses that appear as <code>OR+N</code>.</p><p>One (imperfect) way to try to get a handle on this is to just plot the ratio of <code>OR+N+S</code> to <code>OR+N</code>, because both of these are affected by random drop out.</p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Region</span><span class=o>==</span><span class=s>"England"</span>,<span class=nv>Amplicons</span> <span class=o>%in%</span> <span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=s>"OR+N+S"</span>,<span class=s>"OR+N"</span><span class=o>)</span><span class=o>)</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>, fill<span class=o>=</span><span class=nv>Amplicons</span>,y<span class=o>=</span><span class=nv>Count</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_bar</span><span class=o>(</span>stat<span class=o>=</span><span class=s>"identity"</span>,position<span class=o>=</span><span class=s>"fill"</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span> <span class=o>+</span><span class=nf>labs</span><span class=o>(</span>title<span class=o>=</span><span class=s>"England"</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-7-1.png width=700px style=display:block;margin:auto></p><pre class=chroma><code class=language-r data-lang=r>
<span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span>,<span class=nv>Amplicons</span> <span class=o>%in%</span> <span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=s>"OR+N+S"</span>,<span class=s>"OR+N"</span><span class=o>)</span><span class=o>)</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>, fill<span class=o>=</span><span class=nv>Amplicons</span>,y<span class=o>=</span><span class=nv>Count</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_bar</span><span class=o>(</span>stat<span class=o>=</span><span class=s>"identity"</span>,position<span class=o>=</span><span class=s>"fill"</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span> <span class=o>+</span><span class=nf>labs</span><span class=o>(</span>title<span class=o>=</span><span class=s>"Regions"</span><span class=o>)</span><span class=o>+</span><span class=nf>facet_wrap</span><span class=o>(</span><span class=o>~</span><span class=nv>Region</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-7-2.png width=700px style=display:block;margin:auto></p></div><p>Similarly we can also plot how the proportion of B.1.1.7 incompatible cases (those that do show S amplification) has changed.</p><div class=highlight><pre class=chroma><code class=language-r data-lang=r><span class=nv>data</span> <span class=o>=</span> <span class=nv>data</span> <span class=o>%&gt;%</span> <span class=nf>mutate</span><span class=o>(</span>B117_incompatible <span class=o>=</span> <span class=nf><a href=https://rdrr.io/r/base/grep.html>grepl</a></span><span class=o>(</span><span class=s>'S'</span>,<span class=nv>Amplicons</span><span class=o>)</span><span class=o>)</span>

<span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>Region</span><span class=o>==</span><span class=s>"England"</span>,<span class=nv>Amplicons</span> <span class=o>%in%</span> <span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=s>"OR+N+S"</span>,<span class=s>"OR+N"</span><span class=o>)</span><span class=o>)</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>, fill<span class=o>=</span><span class=nv>B117_incompatible</span>,y<span class=o>=</span><span class=nv>Count</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_bar</span><span class=o>(</span>stat<span class=o>=</span><span class=s>"identity"</span>,position<span class=o>=</span><span class=s>"fill"</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span> <span class=o>+</span><span class=nf>labs</span><span class=o>(</span>title<span class=o>=</span><span class=s>"England"</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-8-1.png width=700px style=display:block;margin:auto></p><pre class=chroma><code class=language-r data-lang=r>
<span class=nf>ggplot</span><span class=o>(</span><span class=nv>data</span><span class=o>%&gt;%</span> <span class=nf><a href=https://rdrr.io/r/stats/filter.html>filter</a></span><span class=o>(</span><span class=nv>RegionType</span><span class=o>==</span><span class=s>"EnglandRegion"</span>,<span class=nv>Amplicons</span> <span class=o>%in%</span> <span class=nf><a href=https://rdrr.io/r/base/c.html>c</a></span><span class=o>(</span><span class=s>"OR+N+S"</span>,<span class=s>"OR+N"</span><span class=o>)</span><span class=o>)</span>,<span class=nf>aes</span><span class=o>(</span>x<span class=o>=</span><span class=nv>week</span>, fill<span class=o>=</span><span class=nv>B117_incompatible</span>,y<span class=o>=</span><span class=nv>Count</span><span class=o>)</span><span class=o>)</span><span class=o>+</span><span class=nf>geom_bar</span><span class=o>(</span>stat<span class=o>=</span><span class=s>"identity"</span>,position<span class=o>=</span><span class=s>"fill"</span><span class=o>)</span><span class=o>+</span><span class=nf>scale_y_continuous</span><span class=o>(</span>labels<span class=o>=</span><span class=nf>scales</span><span class=nf>::</span><span class=nv><a href=https://scales.r-lib.org//reference/label_percent.html>percent</a></span><span class=o>)</span> <span class=o>+</span><span class=nf>labs</span><span class=o>(</span>title<span class=o>=</span><span class=s>"Regions"</span><span class=o>)</span><span class=o>+</span><span class=nf>facet_wrap</span><span class=o>(</span><span class=o>~</span><span class=nv>Region</span><span class=o>)</span>

</code></pre><p><img src=figs/unnamed-chunk-8-2.png width=700px style=display:block;margin:auto></p></div><p>to be continued</p></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=/post/2021-01-22-ons-data/&text=New-variant%20compatibility%20in%20the%20ONS%20infection%20survey" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=/post/2021-01-22-ons-data/&t=New-variant%20compatibility%20in%20the%20ONS%20infection%20survey" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=New-variant%20compatibility%20in%20the%20ONS%20infection%20survey&body=/post/2021-01-22-ons-data/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=/post/2021-01-22-ons-data/&title=New-variant%20compatibility%20in%20the%20ONS%20infection%20survey" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="whatsapp://send?text=New-variant%20compatibility%20in%20the%20ONS%20infection%20survey%20/post/2021-01-22-ons-data/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=/post/2021-01-22-ons-data/&title=New-variant%20compatibility%20in%20the%20ONS%20infection%20survey" target=_blank rel=noopener class=share-btn-weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><img class="avatar mr-3 avatar-circle" src=/authors/admin/avatar_hu8af67025a90ae4f28f8d45441763eced_96138_270x270_fill_q75_lanczos_center.jpg alt="Theo Sanderson"><div class=media-body><h5 class=card-title>Theo Sanderson</h5><h6 class=card-subtitle>Sir Henry Wellcome Fellow</h6><p class=card-text>Biologist developing tools to scale malaria reverse genetics.</p><ul class=network-icon aria-hidden=true><li><a href=/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/theosanderson target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href="https://scholar.google.com/citations?user=voDDwYIAAAAJ" target=_blank rel=noopener><i class="fas fa-graduation-cap"></i></a></li><li><a href=https://github.com/theosanderson target=_blank rel=noopener><i class="fab fa-github"></i></a></li></ul></div></div></div></article></div><div class=page-footer><div class=container><footer class=site-footer></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.js integrity="sha512-SeiQaaDh73yrb56sTW/RgVdi/mMqNeM2oBwubFHagc5BkixSpP1fvqF47mKzPGWYSSy4RwbBunrJBQ4Co8fRWA==" crossorigin=anonymous></script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
        <div class="search-hit-content">
          <div class="search-hit-name">
            <a href="{{relpermalink}}">{{title}}</a>
            <div class="article-metadata search-hit-type">{{type}}</div>
            <p class="search-hit-description">{{snippet}}</p>
          </div>
        </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/en/js/wowchemy.min.297f736e130400781cfee00eff1f1afd.js></script></body></html>