<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Wowchemy 5.0.0-beta.1 for Hugo"><meta name=author content="Theo Sanderson"><meta name=description content="Sometimes in science you come across things that are definitely interesting, and useful, but which you don’t have time to write up properly for one reason or another. I’m going to try to get into the habit of sharing these as blog-posts rather than letting them disappear. Here is one such story.

Not long ago I was searching for orthologs of a malaria gene of interest on the NCBI non-redundant database, which allows one to search across the entire (sequenced) tree of life. Here is a recreation of what I saw:

I was surprised to see that nestled among the Plasmodium species was a sequence from a species called Piliocolobus tephrosceles."><link rel=alternate hreflang=en-us href=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/><link rel=preconnect href=https://fonts.gstatic.com crossorigin><meta name=theme-color content="#1565c0"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.9.0/css/academicons.min.css integrity="sha512-W4yqoT1+8NLkinBLBZko+dFB2ZbHsYLDdr50VElllRcNt2Q4/GSs6u71UHKxB7S6JEMCp5Ve4xjh3eGQl/HRvg==" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin=anonymous media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/github.min.css crossorigin=anonymous title=hl-light media=print onload="this.media='all'"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/styles/dracula.min.css crossorigin=anonymous title=hl-dark media=print onload="this.media='all'" disabled><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.css integrity="sha512-1xoFisiGdy9nvho8EgXuXvnpR5GAMSjFwp40gSRE3NwdUdIMIKuPa7bqoUhLD0O/5tPNhteAsE5XyyMi5reQVA==" crossorigin=anonymous media=print onload="this.media='all'"><script src=https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.2.2/lazysizes.min.js integrity="sha512-TmDwFLhg3UA4ZG0Eb4MIyT1O1Mb+Oww5kFG0uHqXsdbyZz9DcvYQhKpGgNkamAI6h2lGGZq2X8ftOJvF/XjTUg==" crossorigin=anonymous async></script><link rel=preload as=style href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap"><link rel=stylesheet href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&display=swap" media=print onload="this.media='all'"><link rel=stylesheet href=/css/wowchemy.a2dda909fb8228e17412d06b4423e0a6.css><script async src="https://www.googletagmanager.com/gtag/js?id=UA-65298-19"></script><script>window.dataLayer=window.dataLayer||[];function gtag(){dataLayer.push(arguments);}
function trackOutboundLink(url,target){gtag('event','click',{'event_category':'outbound','event_label':url,'transport_type':'beacon','event_callback':function(){if(target!=='_blank'){document.location=url;}}});console.debug("Outbound link clicked: "+url);}
function onClickCallback(event){if((event.target.tagName!=='A')||(event.target.host===window.location.host)){return;}
trackOutboundLink(event.target,event.target.getAttribute('target'));}
gtag('js',new Date());gtag('config','UA-65298-19',{});document.addEventListener('click',onClickCallback,false);</script><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_32x32_fill_lanczos_center_2.png><link rel=apple-touch-icon type=image/png href=/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_192x192_fill_lanczos_center_2.png><link rel=canonical href=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/><meta property="twitter:card" content="summary"><meta property="twitter:site" content="@theosanderson"><meta property="twitter:creator" content="@theosanderson"><meta property="og:site_name" content="Theo Sanderson"><meta property="og:url" content="/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/"><meta property="og:title" content="How I stumbled upon a novel genome for a malaria-like parasite of primates | Theo Sanderson"><meta property="og:description" content="Sometimes in science you come across things that are definitely interesting, and useful, but which you don’t have time to write up properly for one reason or another. I’m going to try to get into the habit of sharing these as blog-posts rather than letting them disappear. Here is one such story.

Not long ago I was searching for orthologs of a malaria gene of interest on the NCBI non-redundant database, which allows one to search across the entire (sequenced) tree of life. Here is a recreation of what I saw:

I was surprised to see that nestled among the Plasmodium species was a sequence from a species called Piliocolobus tephrosceles."><meta property="og:image" content="/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_512x512_fill_lanczos_center_2.png"><meta property="twitter:image" content="/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us"><meta property="article:published_time" content="2018-04-23T17:39:10+00:00"><meta property="article:modified_time" content="2018-04-23T17:39:10+00:00"><script type=application/ld+json>{"@context":"https://schema.org","@type":"BlogPosting","mainEntityOfPage":{"@type":"WebPage","@id":"/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/"},"headline":"How I stumbled upon a novel genome for a malaria-like parasite of primates","datePublished":"2018-04-23T17:39:10Z","dateModified":"2018-04-23T17:39:10Z","author":{"@type":"Person","name":"Theo Sanderson"},"publisher":{"@type":"Organization","name":"Theo Sanderson","logo":{"@type":"ImageObject","url":"/images/icon_hu141e5746c4390abcaff97a6bd8371d9c_4466_192x192_fill_lanczos_center_2.png"}},"description":"\u003cp\u003e\u003cspan style=\"color: gray;\"\u003eSometimes in science you come across things that are definitely interesting, and useful, but which you don’t have time to write up properly for one reason or another. I’m going to try to get into the habit of sharing these as blog-posts rather than letting them disappear. Here is one such story.\u003c/span\u003e\u003c/p\u003e\n\u003chr\u003e\n\u003cp\u003eNot long ago I was searching for orthologs of a malaria gene of interest on the NCBI non-redundant database, which allows one to search across the entire (sequenced) tree of life. Here is a recreation of what I saw:\u003c/p\u003e\n\u003cp\u003e\u003cimg class=\"alignnone size-full wp-image-121\" src=\"/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51.png\" alt=\"\" width=\"780\" height=\"236\" srcset=\"/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51.png 780w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51-300x91.png 300w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51-768x232.png 768w\" sizes=\"(max-width: 709px) 85vw, (max-width: 909px) 67vw, (max-width: 984px) 61vw, (max-width: 1362px) 45vw, 600px\" /\u003e\u003c/p\u003e\n\u003cp\u003eI was surprised to see that nestled among the \u003cem\u003ePlasmodium\u003c/em\u003e species was a sequence from a species called \u003cem\u003ePiliocolobus tephrosceles\u003c/em\u003e.\u003c/p\u003e"}</script><title>How I stumbled upon a novel genome for a malaria-like parasite of primates | Theo Sanderson</title></head><body id=top data-spy=scroll data-offset=70 data-target=#TableOfContents class=page-wrapper><script src=/js/wowchemy-init.min.7264cf0eba3b66951b36da7d2cecf9c5.js></script><aside class=search-modal id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=# aria-label=Close><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search class=form-control aria-label=Search...></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><div class=page-header><nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id=navbar-main><div class=container-xl><div class="d-none d-lg-inline-flex"><a class=navbar-brand href=/>Theo Sanderson</a></div><button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar-content aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none"><a class=navbar-brand href=/>Theo Sanderson</a></div><div class="navbar-collapse main-menu-item collapse justify-content-start" id=navbar-content><ul class="navbar-nav d-md-inline-flex"><li class=nav-item><a class=nav-link href=/#about><span>Home</span></a></li><li class=nav-item><a class=nav-link href=/#featured><span>Publications</span></a></li><li class=nav-item><a class=nav-link href=/#posts><span>Posts</span></a></li><li class=nav-item><a class=nav-link href=/#contact><span>Contact</span></a></li></ul></div><ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2"><li class=nav-item><a class="nav-link js-search" href=# aria-label=Search><i class="fas fa-search" aria-hidden=true></i></a></li><li class="nav-item dropdown theme-dropdown"><a href=# class=nav-link data-toggle=dropdown aria-haspopup=true aria-label="Display preferences"><i class="fas fa-moon" aria-hidden=true></i></a><div class=dropdown-menu><a href=# class="dropdown-item js-set-theme-light"><span>Light</span></a>
<a href=# class="dropdown-item js-set-theme-dark"><span>Dark</span></a>
<a href=# class="dropdown-item js-set-theme-auto"><span>Automatic</span></a></div></li></ul></div></nav></div><div class=page-body><article class=article><div class=article-style><a class=viewsource href=https://github.com/theosanderson/theo.io/tree/master/content/post/archive_posts/2018-04-23-how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes.md>View source</a><div class="article-container pt-3"><h1>How I stumbled upon a novel genome for a malaria-like parasite of primates</h1><div class=article-metadata><span class=article-date>Apr 23, 2018</span>
<span class=middot-divider></span><span class=article-categories><i class="fas fa-folder mr-1"></i><a href=/category/parasites/>Parasites</a></span></div></div><link rel=stylesheet href=/theo.css><div class=article-container><div class=article-style><a class=viewsource href=https://github.com/theosanderson/theo.io/tree/master/content/post/archive_posts/2018-04-23-how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes.md>View source</a><p><span style=color:gray>Sometimes in science you come across things that are definitely interesting, and useful, but which you don’t have time to write up properly for one reason or another. I’m going to try to get into the habit of sharing these as blog-posts rather than letting them disappear. Here is one such story.</span></p><hr><p>Not long ago I was searching for orthologs of a malaria gene of interest on the NCBI non-redundant database, which allows one to search across the entire (sequenced) tree of life. Here is a recreation of what I saw:</p><p><img class="alignnone size-full wp-image-121" src=/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51.png alt width=780 height=236 srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51.png 780w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51-300x91.png 300w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.27.51-768x232.png 768w" sizes="(max-width: 709px) 85vw, (max-width: 909px) 67vw, (max-width: 984px) 61vw, (max-width: 1362px) 45vw, 600px"></p><p>I was surprised to see that nestled among the <em>Plasmodium</em> species was a sequence from a species called <em>Piliocolobus tephrosceles</em>.</p><p>A web search revealed this to be the linnaean name for the <a href=https://en.wikipedia.org/wiki/Ugandan_red_colobus target=_blank rel=noopener>Ugandan red colobus</a> monkey. Odd that this monkey should have a gene so similar to one from a parasite, eh? But there are some weird misannotations on NCBI. However a bit later I saw the same thing happen again with another gene, and thought something must be going on. I tried more malaria genes and found I could almost always find a strong match in the colobus assembly.</p><p>I navigated through NCBI to find some more about this particular monkey.</p><p><img class="alignnone size-full wp-image-122" src=/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.33.54.png alt width=504 height=302 srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.33.54.png 504w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.33.54-300x180.png 300w" sizes="(max-width: 504px) 85vw, 504px"></p><p><a href=https://www.ncbi.nlm.nih.gov/biosample/SAMN07735529/ target=_blank rel=noopener>The page</a> tells us that this monkey from which this genome was sequenced was found here, in the depths of Kibale National Park (it was sequenced for a <a href=http://molecular-anthro.uoregon.edu/index.html target=_blank rel=noopener>University of Oregon</a> genome study, <a href=https://twitter.com/MolecAnth/status/942838967342264320 target=_blank rel=noopener>in prep.</a>):</p><p><img class="alignnone size-full wp-image-123" src=/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.37.08.png alt width=380 height=266 srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.37.08.png 380w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.37.08-300x210.png 300w" sizes="(max-width: 380px) 85vw, 380px"></p><p>It also tells us that the sequence came from whole blood. This methodology seemed to be consistent with researchers serendipitously acquiring sequence from a blood-borne parasite in the process of sequencing the monkey genome.</p><p>Just how much of this genome is there in the colobus sample? I downloaded the assembly to find out. The assembly was broken up into 47,644 contigs. My hypothesis was that some of these represented monkey sequence and others represented this sequence of a malaria-like parasite. I calculated the GC content of each contig and plotted a distribution:</p><p><img class="alignnone size-full wp-image-124" src=/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.55.22.png alt width=398 height=298 srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.55.22.png 398w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-21.55.22-300x225.png 300w" sizes="(max-width: 398px) 85vw, 398px"></p><p><em>Plasmodium spp.</em> have a much lower GC-content than mammalian species, and from this it looks like whatever species we have here shows this same bias. Just to be sure I also searched each “colobus” contig with Diamond (a faster Blast-like tool) against all protein sequences of the rhesus macaque and of <em>P. vivax</em>. If a contig had a significantly higher score in <em>P. vivax</em> than rhesus it was assigned to that species and vice versa. Overlaying this data was pretty compelling:</p><p><img class="alignnone size-full wp-image-125" src=/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-22.07.52.png alt width=754 height=317 srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-22.07.52.png 754w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/Screen-Shot-2018-04-22-at-22.07.52-300x126.png 300w" sizes="(max-width: 709px) 85vw, (max-width: 909px) 67vw, (max-width: 984px) 61vw, (max-width: 1362px) 45vw, 600px"></p><p>I decided that by taking contigs which were either significantly better matches to <em>P. vivax</em>, or alternatively had a GC content below 24%, I would get a genome which was almost entirely this intriguing species. This gave me 13,460 contigs representing about 30 Mbp of sequence (<em>Plasmodium</em> genomes are 20-30 Mbp so this could potentially be a genome’s worth – although I don’t always get BLAST matches, so I suspect it is somewhat less). I also got in touch with the person named in the NCBI accession to see if they were already investigating these:</p><blockquote class=twitter-tweet data-lang=en><p dir=ltr lang=en>hey <a href="https://twitter.com/theosanderson?ref_src=twsrc%5Etfw">@theosanderson</a> not specifically on the hepatocystis seqs in the assembly, though collaborators in our group have done some interesting work on hepatocystis in red colobus themselves <a href=https://t.co/yLs6ycTYrv>https://t.co/yLs6ycTYrv</a></p><p>— Noah Simons (@noahdsimons) <a href="https://twitter.com/noahdsimons/status/981977413314736128?ref_src=twsrc%5Etfw">April 5, 2018</a></p></blockquote><p>My home institute has released a rather neat web tool called <a href=https://companion.sanger.ac.uk/ target=_blank rel=noopener>Companion</a>, which allows you to upload a series of genomic contigs and have them annotated with reference to a known genome using a whole pipeline of analysis. (Unfortunately it only accepts up to 3,000 sequences. But you can download it as a Docker container to circumvent that restriction).</p><p><a href=https://companion.sanger.ac.uk/jobs/dcf60d31de2b43c841a62928 target=_blank rel=noopener>Here</a> are the results from its run on these sequences. Companion identifies >2000 genes, including such favourites as circumsporozoite protein, GAP50, PTEX88 and the SERAs. It nicely annotates a pseudochromosome-based version of the genome (although I’m not sure it’s taking full advantage of the synteny at the moment).</p><p>So what is this mysterious species?</p><p>It clearly isn’t an already sequenced <em>Plasmodium</em> species. A bit of googling suggested to me that it might be some species of <em>Hepatocystis</em>, which are <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3676718/ target=_blank rel=noopener>known to infect red Colobus</a>. Hepatocystis seems in some ways to be an unnecessary genus, since the latest phylogenies show that evolutionarily it lies nested within <em>Plasmodium,</em> being more closely related to mammalian <em>Plasmodium</em> species than is the avian malaria parasite <em>Plasmodium gallinaceum</em>. But there are reportedly important differences of approach between the groups – hepatocystis seems to lack the asexual cycle, with replication occurring only in the liver. One might hypothesise that this would lead to substantial loss of intra-erythrocytic genes. Also these beasts are transmitted by midges rather than mosquitoes (that data the result of a <a href=https://www.cabdirect.org/cabdirect/abstract/19522901249 target=_blank rel=noopener>decade</a>–<a href=https://www.ncbi.nlm.nih.gov/pubmed/13443413 target=_blank rel=noopener>long</a> <a href=https://www.ncbi.nlm.nih.gov/pubmed/13897017 target=_blank rel=noopener>search</a>). There are no other genome-scale sequencing datasets available for hepatocystis.</p><p>I searched GenBank for targeted hepatocystis sequences, and found a few, including the caseinolytic protease C gene which has been sequenced from numerous <a href=https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5537238/ target=_blank rel=noopener>bat</a> <a href=https://www.ncbi.nlm.nih.gov/pubmed/18248741 target=_blank rel=noopener>isolates</a>. I found the matching sequences for these in the Colobus assembly and then Blasted those sequences back against GenBank to recover a set of sequences to analyse. Constructing a tree with <a href=http://www.phylogeny.fr/ target=_blank rel=noopener>Phylogeny.fr</a> seemed to demonstrate clearly that this is some species of hepatocystis:</p><p><img class="alignnone size-full wp-image-141" src=/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/analysis-copy.png alt width=709 height=1146 srcset="/post/archive_posts/old_wp_images/wp-content/uploads/2018/04/analysis-copy.png 709w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/analysis-copy-186x300.png 186w, /post/archive_posts/old_wp_images/wp-content/uploads/2018/04/analysis-copy-634x1024.png 634w" sizes="(max-width: 709px) 85vw, (max-width: 909px) 67vw, (max-width: 984px) 61vw, (max-width: 1362px) 45vw, 600px"></p><p>The closest relative found here is from a <a href=https://www.ncbi.nlm.nih.gov/nuccore/eu254616.1 target=_blank rel=noopener><em>Cynopterus brachyotis</em> bat from Singapore</a> – bats seem to pick up everything!</p><p>This is about as far as I’ve got with this. Some thoughts at this point:</p><ul><li>If you are interested in what a specific gene looks like in hepatocystis it is likely that you can find at least part of it in these colobus contigs to compare to. It is very easy to run <a href=https://www.ncbi.nlm.nih.gov/assembly/1441961 target=_blank rel=noopener>a BLAST search at NCBI to check</a>.</li><li>Early data-release is awesome! Parasitologists will really benefit from the University of Oregon primate researchers making this data available.</li><li>Someone should really see if we can get a really decent genome from these short read sequences. The library was prepared with Chromium technology and assembled with SuperNova, a 10X assembler. Is it possible these methods are optimised for diploid sequences and that assembling the raw data might enable a better parasite assembly? Or maybe the current assembly is pretty good – apart from the gaps – and missing _Plasmodium _genes are truly missing due to the differences in the hepatocystis lifecycle?</li><li>The depositors of the assembly might wish to partition it into its monkey and hepatocystis portions as I have above, lest a lot of parasite researchers are going to get confused when they find their parasite to be so closely related to a monkey 🙂 Happy to share my assignments!</li></ul></div><div class=share-box aria-hidden=true><ul class=share><li><a href="https://twitter.com/intent/tweet?url=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/&text=How%20I%20stumbled%20upon%20a%20novel%20genome%20for%20a%20malaria-like%20parasite%20of%20primates" target=_blank rel=noopener class=share-btn-twitter><i class="fab fa-twitter"></i></a></li><li><a href="https://www.facebook.com/sharer.php?u=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/&t=How%20I%20stumbled%20upon%20a%20novel%20genome%20for%20a%20malaria-like%20parasite%20of%20primates" target=_blank rel=noopener class=share-btn-facebook><i class="fab fa-facebook"></i></a></li><li><a href="mailto:?subject=How%20I%20stumbled%20upon%20a%20novel%20genome%20for%20a%20malaria-like%20parasite%20of%20primates&body=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/" target=_blank rel=noopener class=share-btn-email><i class="fas fa-envelope"></i></a></li><li><a href="https://www.linkedin.com/shareArticle?url=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/&title=How%20I%20stumbled%20upon%20a%20novel%20genome%20for%20a%20malaria-like%20parasite%20of%20primates" target=_blank rel=noopener class=share-btn-linkedin><i class="fab fa-linkedin-in"></i></a></li><li><a href="whatsapp://send?text=How%20I%20stumbled%20upon%20a%20novel%20genome%20for%20a%20malaria-like%20parasite%20of%20primates%20/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/" target=_blank rel=noopener class=share-btn-whatsapp><i class="fab fa-whatsapp"></i></a></li><li><a href="https://service.weibo.com/share/share.php?url=/blog/2018/04/23/how-i-stumbled-upon-a-novel-genome-for-a-malaria-like-parasite-of-apes/&title=How%20I%20stumbled%20upon%20a%20novel%20genome%20for%20a%20malaria-like%20parasite%20of%20primates" target=_blank rel=noopener class=share-btn-weibo><i class="fab fa-weibo"></i></a></li></ul></div><div class="media author-card content-widget-hr"><img class="avatar mr-3 avatar-circle" src=/authors/admin/avatar_hu8af67025a90ae4f28f8d45441763eced_96138_270x270_fill_q75_lanczos_center.jpg alt="Theo Sanderson"><div class=media-body><h5 class=card-title>Theo Sanderson</h5><h6 class=card-subtitle>Sir Henry Wellcome Fellow</h6><p class=card-text>Biologist developing tools to scale malaria reverse genetics.</p><ul class=network-icon aria-hidden=true><li><a href=/#contact><i class="fas fa-envelope"></i></a></li><li><a href=https://twitter.com/theosanderson target=_blank rel=noopener><i class="fab fa-twitter"></i></a></li><li><a href="https://scholar.google.com/citations?user=voDDwYIAAAAJ" target=_blank rel=noopener><i class="fas fa-graduation-cap"></i></a></li><li><a href=https://github.com/theosanderson target=_blank rel=noopener><i class="fab fa-github"></i></a></li></ul></div></div></div></article></div><div class=page-footer><div class=container><footer class=site-footer></footer></div></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i>Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i>Download</a><div id=modal-error></div></div></div></div></div><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/highlight.min.js integrity="sha512-TDKKr+IvoqZnPzc3l35hdjpHD0m+b2EC2SrLEgKDRWpxf2rFCxemkgvJ5kfU48ip+Y+m2XVKyOCD85ybtlZDmw==" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.2.0/languages/r.min.js></script><script src=https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.7.1/leaflet.min.js integrity="sha512-SeiQaaDh73yrb56sTW/RgVdi/mMqNeM2oBwubFHagc5BkixSpP1fvqF47mKzPGWYSSy4RwbBunrJBQ4Co8fRWA==" crossorigin=anonymous></script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
        <div class="search-hit-content">
          <div class="search-hit-name">
            <a href="{{relpermalink}}">{{title}}</a>
            <div class="article-metadata search-hit-type">{{type}}</div>
            <p class="search-hit-description">{{snippet}}</p>
          </div>
        </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script><script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script><script src=/en/js/wowchemy.min.297f736e130400781cfee00eff1f1afd.js></script></body></html>